src/components/select/select.vue

```diff
@@ -212,21 +214,20 @@ useDetectOutsideClick(componentRef, () => {
       id="option__container"
       ref="selectRef"
       :class="selectId"
-      @keydown="handleSearch($event)"
+      @keydown.enter="handleSearch($event)"
     >
       <div
         v-for="(option, index) in optionsList"
         :id="`option-${option.id}`"
         :ref="(el) => { optionRefs[index] = el as HTMLElement }"
         :key="index"
         :class="
-          selectedOption == option.text ? 'option--selected option' : 'option'
+          selectedOption == option.text ? 'option--selected option subtitle2' : 'option subtitle2'
```

Comment:
Since both `option` and `subtitle2` classnames are always present you could put them in a normal class attribute, and only use the conditional for setting the selected option.

Reply:
That is true. Old React habit.